GOVERNANCE.md

```diff
@@ -0,0 +1,204 @@
+range of companies, universities and non-profit organizations. Because of this,
+it is possible that Members will have conflict of interests. Such conflict of
+interests include, but are not limited to:
+
+-   Financial interests, such as investments, employment or contracting work,
+    outside of The Project that may influence their work on The Project.
+-   Access to proprietary information of their employer that could potentially
+    leak into their work with the Project.
+
+All members of the Council shall disclose to the rest of the
+Council any conflict of interest they may have. Members with a conflict of
+interest in a particular issue may participate in Council discussions on that
+issue, but must recuse themselves from voting on the issue.
+
+### Council Selection Process
+
+#### Eligibility
+* Must be core contributor for at least one year
+
+#### Nominations
+* Nominations are taken over a public github issue ticket over the course of 2 weeks.
+* Only Core Contributors may nominate folks
+* Self Nominations are allowed
+* At the conclusion of the 2 weeks, the list of nominations is posted on the ticket and this ticket is closed.
+
+
+#### Election Process
+
+* Voting occurs over 2 weeks, at the conclusion of the nominations. Voting is blind and mediated by either an application or a third party like Numfocus.
+Each voter can vote zero or more times, once per each candidate. As this is not about ranking but about capabilities, voters vote on a yes/no basis per candidate -- “would I trust this person to lead ArviZ?”.
+* In the event of a tie there will be a runoff election for the tied candidates. To avoid further ties and discriminate more among the tied candidates, this vote will be held by [Majority Judgment](https://en.wikipedia.org/wiki/Majority_judgment): for each candidate, voters judge their suitability for office as either Excellent, Very Good, Good, Acceptable, Poor, or Reject. Multiple candidates may be given the same grade by a voter. The candidate with the highest median grade is the winner.
+* If more than one candidate has the same highest median-grade, the MJ winner is discovered by removing (one-by-one) any grades equal in value to the shared median grade from each tied candidate's total. This is repeated until only one of the previously tied candidates is currently found to have the highest median-grade.
+* If ties are still present after this second round, the winner will be chosen at random. Each person tied will pick a number between 0 and 100, and a random integer will be generated from random.org. The person with the closet circular distance will be selected.
+* At the conclusion of voting, all the results will be posted.
+* The decision about who can vote after the first election is deferred to the council. See below for details
+
+#### Length of Tenure and Reverification
+* Council members term limits are 4 years, after which point their seat will come up for reelection.
```

Comment:
Should the council be renewed by half every two years? This could be useful to preserve the know-how the council could have acquired. Probably the election dynamics will make it higly likely that the council updates partially, even if we do not enforce it, but anyway.
It seems the council members can be reelected indefinitely, should this be made explicit?

Reply:
I'm fine with 2 years! If anyone disagrees please make it known now. Council Members can be reelected indefinitely (which is by design but arbitrary an can be changed). I'll make it explicit

Reply:
OK, but my main point is about renewing in halfs or renewing the entire council